checkov/terraform/checks/resource/alicloud/RDSInstanceLogsEnabled.py

```diff
@@ -0,0 +1,22 @@
+from checkov.common.models.enums import CheckCategories,CheckResult
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class RDSInstanceLogsEnabled(BaseResourceCheck):
```

Comment:
why not use your own `AbsRDSParameter` here? 😄

Reply:
yeah sunk cost fallacy there.